File: src/gatling/scala/org/openapitools/client/model/Metadata.scala

package org.openapitools.client.model

import java.time.OffsetDateTime

case class Metadata (
    /* HTTP status of the response (e.g., OK or ERROR) */
    _status: Option[String],
    /* HTTP method of the request */
    _method: Option[String],
    /* Requested URI path */
    _path: Option[String],
    /* Timestamp when response was generated */
    _timestamp: Option[OffsetDateTime],
    _error: Option[ComponentsSchemasMetadataError]
)
object Metadata {
    def toStringBody(var_status: Object, var_method: Object, var_path: Object, var_timestamp: Object, var_error: Object) =
        s"""
        | {
        | "status":$var_status,"method":$var_method,"path":$var_path,"timestamp":$var_timestamp,"error":$var_error
        | }
        """.stripMargin
}

File: client/php-dt/src/App/DTO/Metadata.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

class Metadata
{
    /**
     * HTTP status of the response (e.g., OK or ERROR)
     * @DTA\Data(field="status", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"string"})
     */
    public ?string $status = null;

    /**
     * HTTP method of the request
     * @DTA\Data(field="method", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"string"})
     */
    public ?string $method = null;

    /**
     * Requested URI path
     * @DTA\Data(field="path", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"string"})
     */
    public ?string $path = null;

    /**
     * Timestamp when response was generated
     * @DTA\Data(field="timestamp", nullable=true)
     * @DTA\Strategy(name="DateTime")
     * @DTA\Validator(name="DateTime")
     */
    public ?\DateTimeInterface $timestamp = null;

    /**
     * @DTA\Data(field="error", nullable=true)
     * @DTA\Strategy(name="Object", options={"type":\App\DTO\ComponentsSchemasMetadataError::class})
     * @DTA\Validator(name="TypeCompliant", options={"type":\App\DTO\ComponentsSchemasMetadataError::class})
     */
    public ?\App\DTO\ComponentsSchemasMetadataError $error = null;

}
